src/main/java/cn/nukkit/item/ItemHoneycomb.java

```diff
@@ -0,0 +1,9 @@
+package cn.nukkit.item;
+
+import cn.nukkit.utils.Identifier;
+
+public class ItemHoneycomb extends Item {
```

Comment:
Is this class really needed? I think it could be replaced by a `SimpleItem`, registering like this:
```java
registerVanilla(HONEYCOMB, SimpleItem::new, 736);
```

Reply:
I believe it would work but I saw in other items already implemented that a separate class was created. I believe that for the purpose of "existence record".